composer.json

```diff
@@ -53,7 +53,7 @@
     "require-dev": {
         "ext-pdo": "*",
         "doctrine/coding-standard": "^6.0",
-        "phpstan/phpstan-shim": "^0.9.2",
+        "phpstan/phpstan": "^0.11",
```

Comment:
I don't understand how it was working before… shouldn't it have been `vendor/bin/phpstan.phar` in `.travis.yml`? Also, why this change?

Reply:
`phpstan-shim` is the binary version and I think we're using it to avoid conflicts. I've changed it to align the way we do in other repositores.